Gemfile

```diff
@@ -25,9 +25,17 @@ gem 'bootsnap', '>= 1.4.2', require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 
+gem 'bcrypt', '~> 3.1.7'
+gem 'jwt'
+gem 'rack-cors', :require => 'rack/cors'
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'factory_bot_rails', '~> 4.0'
+  gem 'shoulda-matchers', '~> 3.1'
+  gem 'faker'
+  gem 'database_cleaner'
 end
 
 group :development do
```

Comment:
Could you add ``  gem 'dotenv-rails'`` and `` gem 'wdm', '>= 0.1.0' if Gem.win_platform? `` under :development?
Dotenv is for environment variable processing.
wdm is a gem required for "avoiding polling for changes"_?_ (gem only applied for Windows platforms)